src/app/shared/components/parameters-pair/parameters-pair.component.html

```diff
@@ -17,10 +17,31 @@
       </div>
     </div>
     <div
+      *ngIf="!isCopy"
       ngxClipboard
       class="value"
       [innerHTML]="value"
       [matTooltip]="value"
     ></div>
+    <cs-input-group *ngIf="isCopy" class="value">
+      <mat-form-field class="full-width-input" cs-input-group-textbox>
+        <input matInput [value]="value" readonly>
+      </mat-form-field>
+      <button
```

Comment:
Looks like the button does not fit if Russian language is used and the label is wide